src/core/datastructures/volume/volume.cpp

```diff
@@ -160,6 +162,21 @@ vec3 Volume::getWorldSpaceGradientSpacing() const {
     return ds;
 }
 
+void Volume::setSwizzleMask(const SwizzleMask& mask) {
+    // update swizzle mask of all representations
+    for (auto rep : representations_) {
+        rep.second->setSwizzleMask(mask);
```

Comment:
Im not sure about this, 
seems strange to update it here and then also in the converter functions...